collaboration-and-sharing/public-shareable-links.md

```diff
@@ -0,0 +1,31 @@
+# Create public shareable links
+
+{% hint style="info" %}
+**Enterprise plan only**
+This feature is only available in custom enterprise pricing plans.
+{% endhint %}
+
+When an asset or media collection needs to be shared externally with someone outside your ImageKit environment, a public shareable link can be created for it. Public links allow limited, read-only access to your assets without the need to register for an ImageKit account. This includes the ability to download multiple assets in a ZIP file. A link can be created for a file, folder, or media collection. Configurable options for a public link include password protection, a validity timeframe, and whether to expose [draft assets](../media-library/overview/draft-assets.md) or not. If an expiration date is set for a public link, it will be automatically deleted after expiration.
+
+For restricted media library users, the minimum permission level needed to create, view, and edit public links for an asset or media collection is 'Can manage'. Learn more about permissions and access sharing for restricted users [here](../collaboration-and-sharing/README.md#understanding-permissions-on-assets-and-media-collections).
+
+{% hint style="info" %}
+**Limits**
+For a single asset or media collection, a maximum of 5 public links can be created. Across all assets and media collections in an account, a maximum of 100 public links can be created. This limit is configurable.
+{% endhint %}
+
+You can create a public link for any asset or media collection in your media library.
+
+![Right-click menu location for public links](<../.gitbook/assets/right-click-menu-public-links.png>)
```

Comment:
These images should be put in relevant sections.

Reply:
sections on the page, or sections in the file structure?

Reply:
relocated the images